Add note to "help translate" page (#486)
ScratchAddons/website-v2: content/docs/localization/joining-the-localization-team.md

> If you don't own a Scratch account, please send an email to `translation@scratchaddons.com` instead.
> 
> Do not create a new Scratch account just to fulfill this requirement. Instead, send an email to the address above. New scratcher accounts are ignored.

Reviewer: ```suggestion
**If you don't have a Scratch account**, please send an email to `translation@scratchaddons.com` instead. Do not create a new Scratch account just to fulfill this requirement. New Scratcher accounts will be ignored.
```
